Dockerfile

```diff
@@ -49,6 +49,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
 # Install required ubuntu packages.
 RUN apt-get update && apt-get install -y --no-install-recommends  \
     build-essential       \
+    libpq-dev             \
```

Comment:
Why is this needed now?

Reply:
No idea why it is needed for arm64 only, without this will cause a problem of running aiida-core docker image but not this image. It is header files for PostgreSQL I think.

Reply:
If you still remember what this is needed for in aiida-core, it might be worth writing it in a comment here

Reply:
Add a comment for this, not pretty sure I interpret the error correctly I paste below the error of docker build aiida-core without this `libpq-dev`. 

```
#7 51.98     gcc -pthread -B /opt/conda/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC "-DPSYCOPG_VERSION=2.8.6 (dt dec pq3 ext lo64)" -DPG_VERSION_NUM=130005 -DHAVE_LO64=1 -I/opt/co
nda/include/python3.8 -I. -I/usr/include/postgresql -I/usr/include/postgresql/13/server -c psycopg/adapter_asis.c -o build/temp.linux-aarch64-3.8/psycopg/adapter_asis.o -Wdeclaration-after-statement
#7 51.98     In file included from psycopg/adapter_asis.c:28:
#7 51.98     ./psycopg/psycopg.h:36:10: fatal error: libpq-fe.h: No such file or directory
#7 51.98        36 | #include <libpq-fe.h>
#7 51.98           |          ^~~~~~~~~~~~
#7 51.98     compilation terminated.
#7 51.98     
#7 51.98     It appears you are missing some prerequisite to build the package from source.
#7 51.98     
#7 51.98     You may install a binary package by installing 'psycopg2-binary' from PyPI.
#7 51.98     If you want to install psycopg2 from source, please install the packages
#7 51.98     required for the build and try again.
#7 51.98     
#7 51.98     For further information please check the 'doc/src/install.rst' file (also at
#7 51.98     <https://www.psycopg.org/docs/install.html>).
#7 51.98     
#7 51.98     error: command '/usr/bin/gcc' failed with exit code 1
#7 51.98     ----------------------------------------
#7 51.98 ERROR: Command errored out with exit status 1: /opt/conda/bin/python3.8 -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-825i6nvd/psycopg2-binary_04de1ed0a32a40df8a7edf5b4596a4fe/setup.py'"'"';
 __file__='"'"'/tmp/pip-install-825i6nvd/psycopg2-binary_04de1ed0a32a40df8a7edf5b4596a4fe/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; 
setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-ihv_tvxr/install-record.txt --single-version-externally-managed --compile --inst
all-headers /opt/conda/include/python3.8/psycopg2-binary Check the logs for full command output.
------
error: failed to solve: executor failed running [/bin/sh -c pip install ./aiida-core[atomic_tools]]: exit code: 1
```

Reply:
I think you interpreted correctly - it makes sense that they don't yet have wheels for that architecture